### PR TITLE
Database creation

### PR DESCRIPTION
**Pull request by Rostammahabadi**
Added the following databases with the associations
Senators - no associations
Users - has many house_favorites, has_many house_members, has_many senator_favorites, has_many senators
house_favorites - belongs to house member
senator_favorites - belongs to senator
House Members - no associations

You are able to call user1.senators or user1.house_members and get a collection (array) of senators from their favorites. The helper method will not be hard to implement with what I've made so far.

All tests are passing

### CODE REVIEW

**Review by AntonioJacksonII — APPROVED** [review]
This looks great! Thanks for setting up these relationships.

**Review by dborski — APPROVED** [review]

File: app/models/house_member.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class HouseMember < ApplicationRecord
+
```
Comment (arpariseau) [inline app/models/house_member.rb:2]: May want to have a many_through for later extensions connecting single house members to multiple users (user approval rating of that member?)

File: app/models/senator.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class Senator < ApplicationRecord
+
```
Comment (arpariseau) [inline app/models/senator.rb:2]: Same as house member

File: app/models/house_member.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+end
```
Comment (dborski) [inline app/models/house_member.rb:3]: Leaving a note that in the future we may need to create the relationship with users through the joins table

File: app/models/user.rb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  has_many :house_favorites
+  has_many :house_members, through: :house_favorites
+  has_many :senator_favorites
+  has_many :senators, through: :senator_favorites
+  validates_presence_of :email
```
Comment (arpariseau) [inline app/models/user.rb:6]: Uniqueness?

**Review by arpariseau — APPROVED** [review]

**Pull request merged**